### DISCUSSION TITLE
How to access the Math functions like clamp()

### DISCUSSION THREAD

**Discussion by bpikaar** [excaliburjs/Excalibur#2603] [Q&A] [answered]
I'm using version v0.27.0. How can i reach the clamp() function. I can't import Math.

**Comment by eonarheim** [comment, marked as answer]
Hi @bpikaar 

Great question, `clamp()` is available at the top level import if you are doing commonjs style modules

<img width="574" alt="image" src="https://user-images.githubusercontent.com/612071/225484997-d15262a8-145b-4f91-bfd8-f9db06aae7d6.png">

If you include excalibur.js as a script on the page it will be hanging off of the `ex` global.

```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <script src="https://unpkg.com/excalibur@0.27.0"></script>
    <script src="./main.js"></script>
  </body>
</html>
```

```typescript
// main.js
console.log(ex.clamp(6, 0, 4));
```

Let me know if this helps!

**Reply by bpikaar** [reply]
@eonarheim great stuff thnx. This is my implementation using ES modules but without NPM 

```javascript
import { Engine, clamp } from 'https://esm.sh/excalibur'

export class Game extends Engine {
    constructor() {
        super({ width: 800, height: 600 })

        console.log(clamp(6, 0, 4));

        this.start()
    }
}

new Game()
```
